apps/api/v2/src/ee/bookings/2024-08-13/controllers/bookings.controller.ts

```diff
@@ -209,12 +212,15 @@ export class BookingsController_2024_08_13 {
   }
 
   @Get("/:bookingUid/recordings")
-  @UseGuards(BookingUidGuard)
+  @Pbac(["booking.readRecordings"])
+  @Permissions([BOOKING_READ])
+  @UseGuards(ApiAuthGuard, BookingUidGuard, BookingPbacGuard)
```

Comment:
This is a breaking change - we are requiring auth now and previously were not. Do you confirm that noone is using this endpoint? The breaking changes check pointed it out too correctly.